template/webpack.config.js

```diff
@@ -160,16 +162,27 @@ module.exports = env => {
                         },
                     ].filter(loader => Boolean(loader)),
                 },
+
+                // TODO: Removed the rule once https://github.com/vuejs/vue-hot-reload-api/pull/70 is accepted
+                {
+                    test: /vue-hot-reload-api\/dist\/index\.js$/,
+                    use: "../vue-hot-reload-api-patcher"
+                },
+
                 {
                     test: /\.css$/,
                     use: [
+                        'nativescript-dev-webpack/style-hot-loader',
+                        'css-hot-loader',
```

Comment:
Does the order make a difference? The other rule for `scss` has `css-hot-loader` first, should this be the same?